plugins/inventory/digitalocean.py

```diff
@@ -0,0 +1,194 @@
+# -*- coding: utf-8 -*-
+
+# Copyright: (c), Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+name: digitalocean
+plugin_type: inventory
+author:
+  - Janos Gerzson (@grzs)
+short_description: DigitalOcean Inventory Plugin
+version_added: "0.04"
+description:
+  - DigitalOcean Inventory plugin.
+  - Acquires droplet list from DO api.
+  - Uses configuration file that ends with
+  - (do_hosts|digitalocean|digital_ocean).(yaml|yml)
+extends_documentation_fragment:
+  - community.digitalocean.digital_ocean.documentation
+  - constructed
+  - inventory_cache
+options:
+  plugin:
+    description:
+      - The name of the DigitalOcean Inventory Plugin,
+      - this should always be 'community.digitalocean.digitalocean'.
+    required: True
+    choices: ['community.digitalocean.digitalocean']
+  api_token:
+    description:
+     - DigitalOcean OAuth token.
+    required: True
+    type: str
+    aliases: [ oauth_token ]
+    env:
+      - name: DO_API_TOKEN
+  attributes:
+    description: >-
+      Droplet attributes to add as host vars to each inventory host.
+      Check out the DO api docs for full list of attributes at
+      https://developers.digitalocean.com/documentation/v2/#list-all-droplets
+    type: list
+    default: 'id,name,networks,region,size_slug'
+  var_prefix:
+    description:
+      - Prefix of generated varible names (e.g. 'tags' -> 'do_tags')
+    type: str
+    default: 'do'
```

Comment:
Shoudn't the prefix be `do_` to get `do_tags`? (From a user's point of view, not by looking at the code.)

Reply:
I was thinking of it too but wasn't sure after I saw that the keyed group generator applies an '_' as a separator by default. I don't know what users expect from an inventory plugin. If nobody vote against it within 1-2 days I apply your suggestion for this part as well.

Reply:
A var_separator also could be added, but maybe it's too much...

Reply:
Hmm, right, the keyed group generator also did that... I never understood why. In any case, feel free to do here what you want/prefer.

Reply:
I've chosen your suggestion.